tests/test_node_differ.py

```diff
@@ -328,8 +336,15 @@ def test_moved_section():
     diff = EditTypes(prev_wikitext, curr_wikitext, lang='en').get_diff()
     assert expected_changes == diff
 
-def test_insert_cjk_punctuations():
+def test_change_cjk_punctuations():
     cjk_curr_wikitext = cjk_prev_wikitext.replace('。','、',1)
     expected_changes = {'Section':{'change':1}, 'Punctuation':{'change':1},'Sentence':{'change':1},'Paragraph':{'change':1}}
     diff = EditTypes(cjk_prev_wikitext, cjk_curr_wikitext, lang='ja').get_diff()
+    assert expected_changes == diff
+
+
+def test_remove_cjk_punctuations():
```

Comment:
we had discussed having a test for Japanese that changed some of the words so the non-whitespace-word-delimited languages had a test associated with them. could you either add that or replace this one with a change that affects some words in a sentence?

Reply:
@geohci I have added a test for this.